scripts/prepare-shell.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export PROJECT_DIRECTORY="$(pwd)"
```

Comment:
this requires the caller to know the right cwd to call the script. Consider doing something similar to what genny does:

https://github.com/mongodb/genny/blob/58540d3fc8ed01c16364dfaf254225226c5a364c/run-genny#L10-L13